packages/office-ui-fabric-react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -96,10 +97,26 @@ export class DialogBase extends React.Component<IDialogProps, {}> {
       mergedLayerProps.onLayerDidMount = onLayerDidMount;
     }
 
+    let dialogDraggableClassName: string | undefined;
+    let dragOptions: IDragOptions | undefined;
+
+    // if we are draggable, make sure we are using the correct
+    // draggable classname and selectors
+    if (modalProps && modalProps.dragOptions && !modalProps.dragOptions.dragHandleSelector) {
+      dialogDraggableClassName = 'ms-dialog-draggable-header';
```

Comment:
Nit, but if we're not going to use one of the existing Dialog DOM global selectors, should we at least keep this similar in casing i.e. `ms-Dialog-draggable-header`?